bundler/spec/commands/doctor_spec.rb

```diff
@@ -40,29 +40,39 @@
       allow(File).to receive(:readable?).with(unwritable_file) { true }
     end
 
+    let(:existing_dylib_path) do
```

Comment:
I need some help here. The idea is to really check the existence of 100% existed system libraries but I'm not sure what libraries are 100% present in the OSs.

Reply:
I guess the test could be made more realistic, but I think for now it could do to change the unit test to adapt to the new code, and stub `Fiddle.dlopen` in a similar way we used to stub `File.exist?` before?

Reply:
Yep, fixed and all the commits are squished